native/extensions/assets-manager/AssetsManagerEx.cpp

```diff
@@ -567,7 +571,9 @@ void AssetsManagerEx::downloadVersion() {
     if (versionUrl.size() > 0) {
         _updateState = State::DOWNLOADING_VERSION;
         // Download version file asynchronously
+		#if (CC_PLATFORM != CC_PLATFORM_OPENHARMONY)
         _downloader->createDownloadFileTask(versionUrl, _tempVersionPath, VERSION_ID);
+		#endif
```

Comment:
This file has many `#if`, can this class still works?

Reply:
Should not work properly, the network will be adapted later, and this macro will be removed. The implementation was removed due to possible dependencies on this file